lib/rules/no-warning-comments.js

```diff
@@ -37,6 +37,16 @@ module.exports = {
                     },
                     location: {
                         enum: ["start", "anywhere"]
+                    },
+                    decoration: {
+                        type: ["string", "array"],
+                        items: {
+                            type: "string",
+                            pattern: "^\\S$"
+                        },
+                        pattern: "^\\S$",
+                        minItems: 1,
+                        uniqueItems: true
```

Comment:
Can we simplify this by only accepting arrays?

Reply:
I thought it would be more convenient for the most common case of only ignoring `*`. But I don't feel that strongly about it.